srcpkgs/docker-buildx/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'docker-buildx'
+pkgname=docker-buildx
+_pkgname=buildx
```

Comment:
I would remove _pkgname and spell it out every time, it's not shorter this way.

Reply:
Done in https://github.com/void-linux/void-packages/pull/35682/commits/97bb1195018c9f0d32ea3ec968c4a3e44562e734!